python/lsst/faro/base/CatalogMeasurementBase.py

```diff
@@ -102,6 +101,17 @@ def __init__(self, config, *args, **kwargs):
     def run(self, **kwargs):
         return self.measure.run(self.config.connections.metric, **kwargs)
 
+    def _getTableColumns(self, columns, currentBands=None):
```

Comment:
I think it would be helpful to add a doc string here to explain that the purpose of this function is to add the columns that are needed to apply selections